docs/eventing/getting-started.md

```diff
@@ -1,140 +1,70 @@
 
-### Creating and configuring an Eventing namespace
-
-In this section you create the `event-example` namespace and add a Broker to
-that namespace. You use namespaces to group together and organize your Knative
-resources, including the Eventing subcomponents.
+## Creating a Knative Eventing namespace
 
-1. Run the following command to create a namespace called `event-example`:
-
-   ```sh
-   kubectl create namespace event-example
-   ```
-
-   This creates an empty namespace called `event-example`.
-
-1. Add a Broker named `default` to your namespace with the following command:
-
-   ```sh
-   kubectl create -f - <<EOF
-   apiVersion: eventing.knative.dev/v1
-   kind: Broker
-   metadata:
-     name: default
-     namespace: event-example
-   EOF
-   ```
-
-   This creates the default Broker for the namespace based on config. The Broker
-   is a resource that will allow you to manage your events.
-
-In the next section, you will need to verify that the resources you added in
-this section are running correctly. Then, you can create the rest of the
-eventing resources you need to manage events.
-
-#### Optional Sugar Controller
-
-When installed, the
-[Sugar Controller for Knative Eventing](../install#optional-eventing-extensions)
-will add a controller that reacts to special labels and annotations for your
-resources, you can skip manually creating a Broker in the above step and instead
-label a namespace:
-
-```sh
-kubectl label namespace event-example eventing.knative.dev/injection=enabled
-```
+Namespaces are used to group together and organize your Knative resources.
+<!--TODO: Add documentation about namespaces to core docs?-->
 
-This gives the `event-example` namespace the `eventing.knative.dev/injection`
-label, which adds resources that will allow you to manage your events.
+1. Create a namespace called `event-example` by entering the following command:
 
-### Validating that the `Broker` is running
+  ```
+  kubectl create namespace event-example
+  ```
+  This creates an empty namespace called `event-example`.
 
-The [`Broker`](./broker/README.md#broker) ensures that every event sent by event
-producers arrives at the correct event consumers. The `Broker` was created when
-you labeled your namespace as ready for eventing, but it is important to verify
-that your `Broker` is working correctly. In this guide, you will use the default
-broker.
+## Add a broker to the namespace
 
-1. Run the following command to verify that the `Broker` is in a healthy state:
+The [broker](./broker/README.md#broker) allows you to route events to different event sinks or consumers.
 
-   ```sh
-   kubectl --namespace event-example get Broker default
-   ```
+1. Add a broker named `default` to your namespace by entering the following command:
 
-   This shows the `Broker` that you created:
+  ```
+  kubectl create -f - <<EOF
+  apiVersion: eventing.knative.dev/v1
+  kind: Broker
+  metadata:
+   name: default
+   namespace: event-example
+  EOF
+  ```
```

Comment:
The indentation is off here.

Reply:
Yeah I'm having to do a bit of messing around with it to stop it breaking how the site renders for some reason. I'll get you to take a look over the samples when I feel like I'm finished if that's cool so you can check it's technically correct as well as rendering OK?

Reply:
sounds good!